### DISCUSSION TITLE
Welcome to wezterm Discussions!

### DISCUSSION THREAD

**Discussion by wez** [wezterm/wezterm#363] [General]
## 👋 Welcome!

Let's give the new Discussions feature a try!
We've been using Gitter/Element.io for unstructured discussion and Q&A and I can see some or all of that transitioning here if GitHub's implementation works out.

Please feel empowered to:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

If you'd like to kick things off with an introduction of yourself and the things that you do or would like to do, that'd be welcome!

It's too early for me (@wez) to have strong opinions about how this should be organized, so don't worry about posting in the "wrong" place!

**Comment by tuananh** [comment]
it's already an improvement over Gitter/Element.io since i dont have to login :D
